SPECS/mysql/mysql.spec

```diff
@@ -51,6 +51,12 @@ make DESTDIR=%{buildroot} install
 %check
 make test
 
+%pre
```

Comment:
I was wondering if we should remove this in post, when we un-install ?

Reply:
That's a good idea, added